pkg/api/operators/apis/experiment/v1alpha2/util.go

```diff
@@ -73,10 +73,22 @@ func (exp *Experiment) IsFailed() bool {
 	return hasCondition(exp, ExperimentFailed)
 }
 
+func (exp *Experiment) IsRunning() bool {
+	return hasCondition(exp, ExperimentRunning)
+}
+
+func (exp *Experiment) IsRestarting() bool {
+	return hasCondition(exp, ExperimentRestarting)
+}
+
 func (exp *Experiment) IsCompleted() bool {
 	return exp.IsSucceeded() || exp.IsFailed()
 }
 
+func (exp *Experiment) GetLastConditionType() ExperimentConditionType {
```

Comment:
just for safe, we'd better guard `len(exp.Status.Conditions) > 0` before below

Reply:
@hougangliu Added, with an additional error, if Experiment doesn't have any Condition.